packages/flutter_tools/test/src/testbed.dart

```diff
@@ -33,6 +34,7 @@ final Map<Type, Generator> _testbedDefaults = <Type, Generator>{
       ? FileSystemStyle.windows
       : FileSystemStyle.posix),
   Logger: () => BufferLogger(), // Allows reading logs and prevents stdout.
```

Comment:
Instead of applying this mock everywhere, inject it into the testbed constructor overrides in the testfile:

```
testbed = Testbed(setup: () { ... }, overrides: <Type, Generator>{ ... here })
```

Reply:
Per discussion, this should be in the testbed, but it's a fake, not a mock, so I renamed it as such.